File: code/python/bubblesort.py
def bubble_sort(array):
    array_length = len(array)
    temp = 0
    for i in range(array_length - 1):
        for j in range(array_length - i - 1):
            if array[j] > array[j + 1]:
                temp = array[j + 1]
                array[j + 1] = array[j]
                array[j] = temp
    return array

File: code/C#/BubbleSort.cs
using System;
namespace BubbleSort {
   class MySort {
      static void Main(string[] args) {
         int[] arr = { 21, 50, 12, 31, 61, 40, 30 };
         int temp;
         for (int j = 0; j <= arr.Length - 2; j++) {
            for (int i = 0; i <= arr.Length - 2; i++) {
               if (arr[i] > arr[i + 1]) {
                  temp= arr[i + 1];
                  arr[i + 1] = arr[i];
                  arr[i] = temp;
               }
            }
         }
         Console.WriteLine("Sorted:");
         foreach (int p in arr)
            Console.Write(p + " ");
         Console.Read();
      }
   }
}